authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,14 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from .models import User
 
 """Configure JWT Here"""
+class JWTAuthentication:
+    ''' JWTAuthentication imprements jwt authentication '''
 
```

Comment:
The class docstring is a nice idea. Could you follow the convention of tripple double quotes, and period at the end of the line. [see this](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)